libs9s/s9srpcclient.cpp

```diff
@@ -11476,7 +11472,8 @@ S9sRpcClient::nodesField()
  */
 S9sVariant
 S9sRpcClient::serversField(
-        const S9sVariantList &servers)
+        const S9sVariantList &servers,
+        bool mustHaveSshCredentials)
```

Comment:
Is there any use case which works without ssh credential. If not, I would simplified to one use case